bin/plugin-helpers.js

```diff
@@ -34,12 +34,14 @@ program
   .command('build [files...]')
   .description('Build a distributable archive')
   .on('--help', docs('build'))
+  .option('--skip-package', 'Don\'t create the zip file, leave the build path alone')
```

Comment:
In kibana we call the deb/yum packages "packages", I think we should call this `--skip-archive`